numpy/random/generator.pyx

```diff
@@ -3421,18 +3430,26 @@ cdef class Generator:
         --------
         >>> mean = (1, 2)
         >>> cov = [[1, 0], [0, 1]]
-        >>> x = np.random.default_rng().multivariate_normal(mean, cov, (3, 3))
+        >>> rng = np.random.default_rng()
+        >>> x = rng.multivariate_normal(mean, cov, (3, 3))
         >>> x.shape
         (3, 3, 2)
 
+        We can use a different method other than the default to factorize cov:
+        >>> y = rng.multivariate_normal(mean, cov, (3, 3), method='cholesky')
+        >>> y.shape
+        (3, 3, 2)
+
         The following is probably true, given that 0.6 is roughly twice the
         standard deviation:
 
         >>> list((x[0,0,:] - mean) < 0.6)
         [True, True] # random
 
         """
-        from numpy.dual import svd
+        if method not in {'eigh', 'svd', 'cholesky'}:
+            raise ValueError(
+                "method must to be one of {'eigh', 'svd', 'cholesky'}")
```

Comment:
```suggestion
                "method must be one of {'eigh', 'svd', 'cholesky'}")
```

Alternatively, can we just add this as an `else` below?

Reply:
wouldn't the current approach be better since it avoids the array creation in lines 3455-3456 if the keyword is not valid?